### ISSUE TITLE
Why use MSEloss?

### ISSUE THREAD

**Issue by xjw00654** [WynMew/FaceAttribute#7]
I found your formulate your label as 1 and -1, and how to distinguish the label since MSELoss calculate the loss but dismiss the sign bit?

**Comment by Asunany** [comment]
将loss函数更改为：
loss = nn.CrossEntropyLoss()
，同时标签也做相应的修改，并且Label需要squeeze降维。
